changelogs/CHANGELOG-1.2.md

```diff
@@ -0,0 +1,29 @@
+# Changelog since v1.1.0
+
+## v1.2.0
+
+Date: TBD
+
+### Changes
+
+- Added the check of restic annotation in vsphere plugin to skip pvc snapshot for restic annotated volumes ([#349](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/349), [@lintongj](https://github.com/lintongj))
+- Fix to determine cluster type from vsphere-csi-controller deployment
+  Fix is to watch for vc config secret in vmware-system-csi namespace for vanilla setups.
+  The secret location is the same for wcp setups, no change is necessary.
+  A restart of backup-driver and data-manager is necessary when csi driver is upgraded to 2.3. ([#348](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/348), [@deepakkinni](https://github.com/deepakkinni))
```

Comment:
Restart is only needed for vanilla?

Reply:
@deepakkinni Would you please propose a change log entry for your PR #348? Looks like it is kind of confusing.